src/FileSystem-Tests-Core.package/FileSystemTest.class/instance/testCreateFile.st

```diff
@@ -0,0 +1,8 @@
+tests
+testCreateFile
+	| path |
+	path := FileSystem workingDirectory / 'plonk' / 'griffles'.
+	(FileSystem workingDirectory / 'plonk') ensureCreateDirectory.
```

Comment:
Shouldn't we do these tests in a memory file system to avoid to dirty the user folder containing the image? Or should we also test that it works on a concrete FS?